ios/Plugin/SimPlugin.swift

```diff
@@ -7,12 +7,19 @@ import Capacitor
  */
 @objc(SimPlugin)
 public class SimPlugin: CAPPlugin {
-    private let implementation = Sim()
 
-    @objc func echo(_ call: CAPPluginCall) {
-        let value = call.getString("value") ?? ""
+    private let implementation: Sim = {
```

Comment:
If the intention is to run some code on plugin load, you can use the load() method, just like Android does, see example [here](https://github.com/RRGT19/capacitor-sim-plugin/blob/main/android/src/main/java/com/capacitorjs/plugins/sim/SimPlugin.java#L15) and docs [here](https://capacitorjs.com/docs/plugins/ios#running-code-on-plugin-load) for ios.

_Although is not mandatory, but maybe would be good to keep the style on both platforms._